server/routes/handlers/users.js

```diff
@@ -1,10 +1,29 @@
+const { find } = require('lodash');
 const userController = require('../../controllers/users');
 const authController = require('../../controllers/auth');
 
 module.exports = {
   post: [
     (req, res, next) =>
       userController
+        .getAll({ email: req.body.email }, ['auth'])
```

Comment:
My understanding of not restricting "by query" to the "basic" role is that you allow users to register if their email is already used as a user but not having an auth.
Am i right ?

Reply:
It's giving me all the matching users whether they have an auth or not anyway, and I wanted to check if any admins have the same email, which querying for basic role wouldn't give me.